composer.json

```diff
@@ -7,15 +7,15 @@
   "prefer-stable": true,
   "require": {
     "php": "^8.1.0",
-    "symfony/console": "6.2.*"
+    "symfony/console": "^5.4",
+    "symfony/http-kernel": "^5.4",
+    "symfony/config": "^5.4",
+    "symfony/dependency-injection": "^5.4"
   },
   "require-dev": {
     "phpstan/phpstan": "^1.10",
     "phpunit/phpunit": "^9.5",
-    "friendsofphp/php-cs-fixer": "^3.1",
```

Comment:
Why? because `friendsofphp/php-cs-fixer` has Symfony dependencies, so to avoid locking Symfony version due to "dev tools", the tool is located in a dedicated directory.